### ISSUE TITLE
[Feature Request] Change language

### ISSUE THREAD

**Issue by rampionx** [IgnisDa/ryot#684]
Following our exchanges in the Discussions section, would it be possible to consider translating the app? I am French, and I would like the media titles and their synopses to be in French, as well as the interface, menus, and buttons. Below is a screenshot of the bonukai/MediaTracker app:
![image](https://github.com/IgnisDa/ryot/assets/10504691/59c99979-93bc-4ec0-804b-ff3d4bcb7b47)

**Comment by IgnisDa** [comment]
Yep. Are you willing to contribute this feature?

**Comment by rampionx** [comment]
> Yep. Are you willing to contribute this feature?

Yes with pleasure. I can help translate the interface into French. However, I have no dev skills for the media part.

**Comment by IgnisDa** [comment]
I will look into internationalization.

**Comment by diogoan** [comment]
@IgnisDa I can help translating into Portuguese.

**Comment by DickenSerm** [comment]
@IgnisDa I see a lot of potential in this project, so I would like to help translate it into German.

**Comment by IgnisDa** [comment]
Notes to self:

- DO NOT USE PATHLESS ROUTES!!!! OR ELSE YOU WILL FUCK UP THE ENTIRE ROUTE TREE!!!!
- Store language in preferences.
- Look into performance hit. If there is one, find caching strategies to mitigate.

**Comment by mattuylee** [comment]
is there any update? Maybe I can help translating into Simplified Chinese~

Btw I think localized media content (titles and their synopses, and so on) is much more important than interface.

**Comment by IgnisDa** [comment]
Localized content should be easier that i18n of the app itself. I will take a look at this weekend.

**Comment by IgnisDa** [comment]
A large part of this is now live in Ryot v10. You can choose languages for providers without fiddling with environment variables. Next stop: i18n of the app itself!

**Comment by NicolasGoeddel** [comment]
I would like to help with German.

**Comment by cairobraga** [comment]
> [@IgnisDa](https://github.com/IgnisDa) I can help translating into Portuguese.

I can also do Portuguese, BR and PT, and also Spanish (ES).
